src/interfaces/ICanonGuard.sol

```diff
@@ -124,7 +124,7 @@ interface ICanonGuard is ISafeManageable {
   error TxExpiryDelayCannotBeGreaterThanMax();
 
   /**
-   * @notice Thrown when the long transaction execution delay is greater than the maximum value (uint128.max)
+   * @notice Thrown when the long transaction execution delay is greater than the maximum value (6 months)
```

Comment:
```suggestion
   * @notice Thrown when the long transaction execution delay is greater than the maximum value
```

Having the value there can be easily outdated if there is any change to the constant